acme/authority.go

```diff
@@ -54,8 +57,24 @@ var (
 	orderTable             = []byte("acme_orders")
 	ordersByAccountIDTable = []byte("acme_account_orders_index")
 	certTable              = []byte("acme_certs")
+	ordinal                int
 )
 
+// Ordinal is used during challenge retries to indicate ownership.
```

Comment:
We should document what the init method does (what is it's purpose) above the method. Is it meant to initialize an authority? Is it meant to initialize just the "ordinal"?

Note: we generally don't document unexported methods unless they are doing something worth documenting. However, we do ty to use descriptive names. `init` could refer to a lot of things.

Reply:
I thought `init` was how you run go code at package import time. Or is it just any func starting with `init`?

Reply:
No, I think that's right. An init method gets run when the package is loaded. However, it would be nice to document what the method is actually responsible for doing. Especially in cases where the name of the method/func can't be very descriptive.

Reply:
I mention that the ordinal is used during challenge retries to indicate ownership and there's extensive commentary on a few of the methods where the meat of the usage is. Are you just looking for more context here? Or something more like "Load ordinal from the environment and store it in a var" type of comment?

Reply:
Yes, it's to run code at import time.